express_relay/sdk/js/src/index.ts

```diff
@@ -409,27 +492,67 @@ export class Client {
     };
   }
 
+  private toServerOpportunityBid(
+    bid: OpportunityBid
+  ): components["schemas"]["OpportunityBid"] {
+    return {
+      amount: bid.bid.amount.toString(),
```

Comment:
does the type generation have a way to auto convert to json format? or do like json.stringify or something? then you don't have to rewrite this whenever the type changes in versions

Reply:
They just [suggest](https://openapi-ts.pages.dev/advanced#embrace-snake-case-in-js) using snake_case everywhere but it's a bit too late for that. I guess anything magical here would remove the current type safety checks we have. Do you have anything in mind?

Reply:
nit: I think if the type is identical to server type you can consider using https://quicktype.io/ to generate types with conversion based on the JSON Schema of the API.